examples/arduino/README.md

```diff
@@ -0,0 +1,69 @@
+# Arduino Example
+
+A simple example showing how to use the Notecard Environment Variable Manager (NEVM) with Arduino.
+
+## `arduino.ino`
+
+This file contains the example code. Before building it, you'll first need to uncomment this line:
+
+```c
+// #define PRODUCT_UID "com.your-company:your-product-name"
+```
+
+and replace `com.your-company:your-product-name` with [the ProductUID](https://dev.blues.io/notehub/notehub-walkthrough/#finding-a-productuid) from a [Notehub](https://dev.blues.io/notehub/notehub-walkthrough/) project.
+
+The `arduino.ino` code sends a request to Notehub every 20 seconds for these environment variables:
+
+- `variable_a`
+- `variable_b`
+- `variable_c`
+
+## Notehub
+
+Navigate to your [Notehub project](https://notehub.io/projects), click the Devices tab, double-click your device, and open the Environment tab. Under "Device environment variables", set a value for each variable and click Save:
+
+![Settings variables on Notehub](../images/setting_vars_on_notehub.png "Settings variables on Notehub")
+
+## Hardware
+
+We'll be using [the Swan](https://dev.blues.io/swan/introduction-to-swan/) as our target MCU. You will also need
+
+- Micro USB cable
+- Notecard
+- Cellular antenna
+- Notecarrier F
+
+You can get the Notecard, antenna, and Notecarrier F all in one bundle with the [Blues Starter Kit](https://shop.blues.io/products/blues-starter-kit-for-north-america)
```

Comment:
Use this link instead, so folks from Europe, Middle East and Africa can choose the relevant starter kit.

```suggestion
You can get the Notecard, antenna, and Notecarrier F all in one bundle with the [Blues Starter Kit](https://shop.blues.io/collections/blues-starter-kits)
```